src/main/webapp/view/discussion/discussion-comment.js

```diff
@@ -66,6 +67,7 @@ export default class DiscussionComment extends HTMLElement {
     this.setHeader();
     this.setContent();
     this.setTypeTag(comment.type);
+    this.setMarkAsButton(comment.type);
```

Comment:
What does "markAsButton" mean? Is it indicating it needs a button? Would it be better to be isButton(boolean)?

Reply:
This function sets the text and event handler of the "Mark as Answered"/"Mark as Unanswered" button for question-type comments

Reply:
what about setIsAnsweredButton? setMarkAsButton is vague and not easy to understand from the name.